rpc-client-nonce-utils/src/blockhash_query.rs

```diff
@@ -33,11 +33,9 @@ impl Source {
                     .value;
                 Ok((res.0, res.1))
             }
-            Self::NonceAccount(ref pubkey) => {
-                #[allow(clippy::redundant_closure)]
-                let data = crate::get_account_with_commitment(rpc_client, pubkey, commitment)
-                    .and_then(|ref a| crate::data_from_account(a))?;
-                Ok((data.blockhash(), data.fee_calculator))
+            Self::NonceAccount(ref _pubkey) => {
+                // Note - to break or to gut entire deprecated function?
+                Err("Deprecated, NonceAccount no longer support fee_calculator".into())
             }
```

Comment:
Is it time to remove deprecated `get_blockhash_and_fee_calculator()` function, (and one more below)? Or changes here to return Err is acceptable?